<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="graphPlotter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="fromInput">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>104</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="toInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>104</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="from">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FROM</string>
    </property>
   </widget>
   <widget class="QLabel" name="to">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPlot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPlot">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
